### PR TITLE
Ruby Group Capstone

### PR DESCRIPTION
**Pull request by MustakimMasum**
Activities of this group capstone project

- Created item class
- Implemented can_be_archived method
- Implemented move_to_archive method
- Ceare main.rb file
- Created book, music album, game, and movie classes
- Created add_item, can_be_archived methods
- Created the app UI
- Created the data saving and loading functionality
- Added unit tests
- Recorded a project presentation video [Click here to watch the video](https://www.loom.com/share/672a61a869784be587c64fe6cb3e4fc9?sid=d4291f81-4446-47f9-8cbe-cae39eab8cc4)

Kindly asking for a review

### CODE REVIEW

File: spec/label_spec.rb
Comment (fobadara) [inline spec/label_spec.rb:1]: [REQUIRED] I  get some errors when I try to run specs for label
image

File: spec/game_spec.rb
Comment (fobadara) [inline spec/game_spec.rb:1]: [REQUIRED] I get some failing tests in game spec
image

File: spec/book_spec.rb
Comment (fobadara) [inline spec/book_spec.rb:1]: [REQUIRED] An error occurs when I try to run book spec
image

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

REDO

## Highlights
✅ Descriptive README
✅ Descriptive PR summary
✅ Clean code
✅ Good first attempt
✅ Good implementation of the principle of separating concerns.
✅ Good file structure
✅ Beautiful design
✅ Nice implementation of the application through the use of options

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
[the remark this answers is not in the document]
Comment (boniface193) [reply README.md:1]: what do you mean by a video description?

**Review by uwadonat — APPROVED** [review]
## Hi @Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Good project structure ✔️
- Detailed commit message ✔️
- No linter errors ✔️

approval

 
# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**